index.rst

```diff
@@ -16,6 +16,27 @@ metadata counterpart of miniSEED is `StationXML <http://docs.fdsn.org/projects/s
 which is used to describe characteristics needed to interpret the data
 such as location, instrument response, etc.
 
+----------
+Versioning
+----------
+
+Starting with version 3 of **miniSEED**, the specification version is a single
+integer. Any non-backward compatible change to the structure of the header or
+record results in an increment to this number. There are no minor revisions.
+However, the addition of new data encodings or new reserved headers is
+considered backwards compatible. The FDSN may add encodings and extra headers
+within the FDSN key or make editorial changes to the specification without change to the
+major version, resulting only in a new revision of the specification document.
+Note that all parsing software MUST allow for the potential of unknown, new data
+encodings and new extra headers as these are subject to extension. This
+requirement allows older software to parse future records successfully within
+the same specification major version, even though they may be unable to
+decompress the data payload for new encodings.
+
+.. BUILD: remember to also change in conf
+This document is miniSEED 3 as revised 2023-01-17 (DRAFT).
```

Comment:
I'm not very familiar with this environment so I have to ask whether it's feasible to avoid needing to specify the specification version in more than one place. For example, since _conf.py_ is evaluated at runtime, can the version/revision info be written somewhere that can be picked up and rendered this file?

Reply:
Would be nice if it were automated and seems like something Sphinx would know how to do, @chad-earthscope do you know if/how this could be done?

Reply:
> Would be nice if it were automated and seems like something Sphinx would know how to do, @chad-earthscope do you know if/how this could be done?

I've added code to this PR to use the centralized version information in conf.py.  I'm good with merging but would ask someone else review before doing so.

PS for what it's worth I don't think many people know what Sphinx can do, and Sphinx + RST + HTML + Python == zero fun, everything feels like a kludge. /minirant